apps/khix/src/app/globals.css

```
@@ -209,10 +211,10 @@ body {
 .khix-logo-lockup {
   position: relative;
   isolation: isolate;
-  width: min(46vw, 30rem);
-  filter: drop-shadow(0 0 0.45rem rgba(255, 255, 255, 0.92))
+  width: min(60vw, 40rem);
+  /*filter: drop-shadow(0 0 0.45rem rgba(255, 255, 255, 0.92))
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)